README.md

```diff
@@ -134,12 +134,18 @@ identifiers.
 Configure SwiftLint by adding a `.swiftlint.yml` file from the directory you'll
 run SwiftLint from. The following parameters can be configured:
 
+Rule inclusion:
```

Comment:
I'm sure there is probably a better way to list these rules, I'm bad at describing things :(